src/config/sue/default.ts

```diff
@@ -55,6 +55,8 @@ const requiredFieldsDefaults = {
   }
 };
 
+const showFeedbackSection = false;
```

Comment:
should we name it `showFeedbackSectionDefault` to match the others?

Reply:
Yes, that's right. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/150eb625b1f2e7b261cae293fe76b05a6c4aa6e3